index.html

```diff
@@ -4152,6 +4152,105 @@ <h2>Presenting Time Values</h2>
 
     </section>
 
+    <section>
```

Comment:
I think the change you made above (to replace the string "hello world" with a data representation) addresses our (pending) comment. 

I'm not sure I would add an I18N Considerations section. (Generally I18N recommends writing a section like this only when it goes beyond "eat your vegetables".) While there isn't really anything wrong with the text here, it's somewhat orthogonal to this specific spec's task. Implementers will need to read the other VC specs anyway, where they will find similar guidance in a more directly-relevant context. 

So... unless you disagree, I would just drop this?

Reply:
Ok, I'll drop the section before merging this.

Reply:
Done in https://github.com/w3c/vc-data-integrity/pull/213/commits/57f0e78a2a8b5713e53c4af5307a3c60a823350f